common/djangoapps/philu_commands/management/commands/auto_generate_certificates_for_open_courses.py

```diff
@@ -45,16 +54,25 @@ def handle(self, *args, **options):
                 user = user_course_enrollment.user
                 cert_data = _get_cert_data(user, course, user_course_enrollment.mode)
 
-                if not cert_data or cert_data.cert_status != CertificateStatuses.requesting:
+                course_chapters = modulestore().get_items(
+                    course.id,
+                    qualifiers={'category': 'course'}
+                )
+                COURSE_STRUCTURE_INDEX = 0
+                today = datetime.now(utc).date()
+                delta_days = (today - user_course_enrollment.created.date()).days
+                total_modules = len(course_chapters[COURSE_STRUCTURE_INDEX].children)
+                is_certificate_generated = GeneratedCertificate.objects.filter(course_id=course.id, user=user).exists()
```

Comment:
This check is already there and not needed.

Reply:
removed!!